lib/ebae_web/views/layout_view.ex

```diff
@@ -1,3 +1,9 @@
 defmodule EbaeWeb.LayoutView do
   use EbaeWeb, :view
+
+  alias Ebae.Accounts.Guardian
+
+  def signed_in?(conn) do
+    Guardian.Plug.authenticated?(conn)
```

Comment:
This seems off. Are you getting `Guardian.Plug` from `Ebae.Accounts.Guardian`?

Reply:
Yes, trying to `import Guardian` here instead causes failures because it cannot find the required methods configured by `Ebae.Accounts.Guardian`